src/main/java/com/conveyal/datatools/manager/jobs/validation/SharedStopsValidator.java

```diff
@@ -127,17 +127,19 @@ public void validate() {
 
                 // Check for SS_01 (stop id appearing in multiple stop groups).
                 // Make sure this error is only returned if we are inside the feed that is being checked.
-                if (seenStopIds.contains(stopId)) {
-                    if (feedId.equals(sharedStopFeedId)) {
+                Stop syntheticStop = new Stop();
+                syntheticStop.stop_id = stopId;
+                if (feedId.equals(sharedStopFeedId)) {
+                    if (seenStopIds.contains(stopId)) {
                         registerError(stops
```

Comment:
I'm pretty sure this can be simplified to the following because `syntheticStop` equals `stopId`:

`registerError(syntheticStop, NewGTFSErrorType.MULTIPLE_SHARED_STOPS_GROUPS);`

Reply:
We prefer not using synthetic stop if possible, since synthetic stop can't link to the editor